lab_1_keywords_tfidf/main.py

```diff
@@ -157,11 +254,33 @@ def calculate_expected_frequency(
         dict[str, float] | None: Dictionary with expected frequencies.
         In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(doc_freqs, dict) or not isinstance(corpus_freqs, dict):
+        return None
+    for key in doc_freqs:
+        if not isinstance(key, str):
+            return None
+    if doc_freqs == {}:
+        return None
+    expected_frequency = {}
+    words_in_doc = sum(doc_freqs.values())
+    words_in_corpus = sum(corpus_freqs.values())
+    for word in doc_freqs:
+        w_in_doc = doc_freqs[word]
+        w_in_corp = corpus_freqs.get(word, 0)
+        wo_w_in_d = words_in_doc - w_in_doc
+        wo_w_in_c = words_in_corpus - w_in_corp
+        expected = (
+            (w_in_doc + w_in_corp)*(w_in_doc + wo_w_in_d)/(w_in_doc + w_in_corp + \
```

Comment:
very hard to read, try to use brackets istead of slashes to move text to another line